src/types/reader.types.ts

```diff
@@ -25,9 +26,19 @@ export enum Menu {
   open = "open",
   close = "close"
 }
+
+export type AutoLoadChapter = boolean;
```

Comment:
why did you need to define a type for this

Reply:
It's part of the settings definition and just makes sense to define it like the others

Reply:
i think thats overdoing it. it makes sense for the other ones since they are enums, but i think
```
autoLoadChapter: boolean
```
is way more clear than
```
autoLoadChapter: AutoLoadChapter
```

Reply:
Sure